src/AkamaiClient.php

```diff
@@ -426,8 +426,10 @@ public function removeInvalidUrls($urls) {
     $urls_to_clear = [];
     $base_url = $this->drupalConfig->get('basepath');
     foreach ($urls as $path) {
-      if ($path[0] === '/') {
-        $path = ltrim($path, '/');
+      if (is_array($path)) {
```

Comment:
I don't understand this change? Aren't we iterating through an array of strings, so `$path` should be a string?